infrastructure/src/main/java/org/corfudb/infrastructure/FailureHandlerDispatcher.java

```diff
@@ -54,10 +56,10 @@ public void dispatchHandler(IFailureHandlerPolicy failureHandlerPolicy, Layout c
      * Seals the epoch
      * Set local epoch and then attempt to move all servers to new epoch
      *
-     * @param layout Current layout to be sealed
+     * @param currentLayout Current layout to be sealed
      */
-    private void sealEpoch(Layout layout) {
-        layout.setEpoch(layout.getEpoch() + 1);
-        layout.moveServersToEpoch();
+    private void sealEpoch(Layout currentLayout) throws QuorumUnreachableException {
+        currentLayout.setEpoch(currentLayout.getEpoch() + 1);
```

Comment:
By the way, shouldn't the epoch increment happen as a part of the new layout construction and not part of seal?